source/sync/configure/enable-sync.txt

```diff
@@ -219,5 +213,21 @@ Procedure
                curl https://services.cloud.mongodb.com/api/admin/v3.0/groups/{GROUP_ID}/apps/{APP_ID}/services/{MongoDB_Service_ID}/config \
                  -X PATCH \
                  -h 'Authorization: Bearer <Valid Access Token>' \
-                 -h "Content-Type: application/json"
-                 -d @/sync/config.json
+                 -h "Content-Type: application/json" \
+                 -d '<Flexible Sync Configuration>'
+
+            .. code-block:: json
+
+               {
+                 "flexible_sync": {
+                   "state": "enabled",
+                   "database_name": "<Name of Database>",
```

Comment:
we should make it clear that the `database_name` field is only applicable when Development Mode is enabled

Reply:
This is a template for possible fields; I think the user should view the configuration file for more details